kolibri/plugins/facility/assets/src/views/DataPage/index.vue

```diff
@@ -168,6 +176,14 @@
       pollForTasks() {
         return this.$route.name === PageNames.DATA_EXPORT_PAGE;
       },
+      windowSizeStyle() {
+        if (this.windowIsMedium || this.windowIsSmall) {
+          return {
+            float: 'left',
```

Comment:
Floats can be quite buggy across browsers. Based off what I see on [Can I Use](https://caniuse.com/?search=order), you could use `order` which I think is a bit more accurate to the intent here.

With either `float` or `order`, I think there's an important note regarding accessibility in the [MDN `order` docs](https://developer.mozilla.org/en-US/docs/Web/CSS/order):
> Since order is only meant to affect the visual order of elements and not their logical or tab order...

> Using the order property will create a disconnect between the visual presentation of content and DOM order. This will adversely affect users experiencing low vision navigating with the aid of assistive technology such as a screen reader. If the visual (css) order is important, then screen reader users will not have access to the correct reading order.

I don't think the DOM order is all that important, but specifically the keyboard navigable ordering seems important. The [linked article](https://tink.uk/flexbox-the-keyboard-navigation-disconnect/) has some recommendations.

Reply:
This looks interesting and you make a good point about the keyboard navigation, I will try implementing `order` instead and look into the recommendations listed in the article.